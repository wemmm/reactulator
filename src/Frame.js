import React from 'react';

class Frame extends React.Component {

  render() {
    return (
      <div className="frame">
        <div className="calculator-title">
          Reactulator
        </div>
      </div>
    );
  }
}

export default Frame;
