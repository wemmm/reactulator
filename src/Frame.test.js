import React from 'react';
import ReactDOM from 'react-dom';
import Frame from './Frame';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Frame />, div);
});

it('contains specific text', () => {
  const component = renderer.create(
    <Frame />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
