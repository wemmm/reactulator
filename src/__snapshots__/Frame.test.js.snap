// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contains specific text 1`] = `
<div
  className="frame"
>
  <div
    className="calculator-title"
  >
    Reactulator
  </div>
</div>
`;
